cmd/system-probe/main.go

```diff
@@ -124,14 +126,6 @@ func main() {
 	}
 	defer sysprobe.Close()
 
-	platform, err := util.GetPlatform()
-	if err != nil {
-		log.Debugf("error retrieving platform: %s", err)
-	} else {
-		log.Infof("running on platform: %s", platform)
-	}
-
-	log.Infof("running system-probe with version: %s", versionString(", "))
 	go sysprobe.Run()
 	log.Infof("system probe started")
```

Comment:
Small change but while we're here can we change this to `system probe successfully started`? I think it'll help for debugging on the customer's side.